### ISSUE TITLE
ignoring ExtensionTargets when loading VS Solutions

### ISSUE THREAD

**Issue by Matthew-Rennie** [slynch8/10x#2513]
Hey, we have started trailing 10x but are encountering a issue with loading visual studio projects.

as part of our build process, we have some files that are run though our in-house compiler, this is specified in our .vcxproj files with this block of xml, inside xyz.targets we specify what visual studio should do with files that have the .xyz file extension
```
<ImportGroup Label="ExtensionTargets">
  <Import Project="xyz.targets" />
</ImportGroup>
```

Files are then included in the project like with .h/.cpp files, but using our custom tag XYZCompile, which was specified in the above .targets file telling VS to use our build tool
```
<ItemGroup>
   <XYZCompile Include="resource.xyz" />
</ItemGroup>
```

As seen here, the .xyz file is shown in the VS solution explorer but is not being shown by 10x in the workspace tree or with the FindFile command
![image](https://github.com/slynch8/10x/assets/36052246/68b60377-51c9-4a2b-874d-9a4f85a68396)

I've included a project with a setup the reproduces this issue
[10xSolutionBug.zip](https://github.com/slynch8/10x/files/14624511/10xSolutionBug.zip)

**Comment by slynch8** [comment]
Thanks for the test project, that was really helpful.
Here's a fix, let me know if it works on your work project:
https://e.pcloud.link/publink/show?code=XZfa59ZrtKJYpCeUfY9yBenweS3c7C7BNy7

**Comment by Matthew-Rennie** [comment]
unfortunately I'm unable to access that link on my work PC, so I wont be able to test it properly until the change makes its way into an early access build, thanks for the quick fix

**Comment by slynch8** [comment]
I've uploaded a fix (1.0.137). You'll need to enable EarlyAccessUpdates to get it.

**Comment by Matthew-Rennie** [comment]
Verified fixed v1.0.138
